apps/backend/src/app/dto/map/map-zones.dto.ts

```diff
@@ -224,14 +203,16 @@ export class MapZonesDto /* extends JsonifiableDto */ implements MapZones {
   @IsInt()
   readonly formatVersion: number;
 
-  @NestedProperty(TracksDto, { required: true })
-  readonly tracks: Tracks;
-
-  @NestedProperty(VolumeDto, {
-    required: true,
-    isArray: true
+  @ApiProperty({
+    description: 'The sv_maxvelocity value set on the track',
```

Comment:
This is map wide rather than per track now.

Reply:
This still needs to be resolved.

Reply:
Woops sorry, thanks for double checking this!

Reply:
Fixed now